spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -228,7 +231,7 @@ spring:
             - GET    /tasks/info/*                   => hasRole('ROLE_VIEW')
 
             - GET    /tasks/thinexecutions           => hasRole('ROLE_VIEW')
-
+            - GET    /tasks/thinexecutions/*         => hasRole('ROLE_VIEW')
```

Comment:
This looks like a bug fix.  This probably needs to be in a separate PR with an explanation as to why it is needed.

Reply:
Yes please